core/src/main/java/com/novoda/noplayer/internal/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -322,4 +322,14 @@ float getVolume() {
         assertIsInPlaybackState();
         return volume;
     }
+
+    void clearMaxVideoBitrate() {
+        assertIsInPlaybackState();
+        NoPlayerLog.w("Tried to clear max video bitrate but has not been implemented for MediaPlayer.");
```

Comment:
🤷‍♂️ nothing we can do about it, I used the same way of no-opping we used in other methods in `AndroidMediaPlayerFacade`.

Reply:
It's going to be deprecated soon enough.